test/config-next/nonce.json

```diff
@@ -0,0 +1,20 @@
+{
+    "NonceService": {
+        "syslog": {
+            "stdoutLevel": 6
+        },
+        "debugAddr": ":8111",
+        "grpc": {
+            "address": ":9101",
+            "maxConcurrentStreams": 2000,
```

Comment:
I think we should eliminate maxConcurrentStreams: https://github.com/letsencrypt/boulder/issues/4242

Reply:
Do we already have the gRPC version with the fix or is this a general comment not to be addressed in this PR?

Reply:
We already have the gRPC version with the fix, so this comment is requesting to remove maxConcurrentStreams from this PR since we're going to sweep through and delete it everywhere soon.